clang/test/CIR/CodeGen/switch.cpp

```diff
@@ -275,14 +276,124 @@ void sw12(int a) {
 // CHECK-NEXT:       cir.break
 // CHECK-NEXT:     }
 
-void fallthrough(int x) {
+void sw13(int a) {
```

Comment:
These testcases are great. Can you add a couple more variations where extra nested switch shows up? Just wanna make sure the lexical scope here is doing the right thing.